### PR TITLE
#4566 Hover effect on links in navbar

### LINKED ISSUE #4566: Hover effect on links in navbar
image
- missing hover effect on links
image
should be: 
black to k-hovergrey (#6B6B6B)
white to k-shade

more here: 
- #4456

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

## Context

- [x] Closes #4566
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: 
[Payout](https://beta.kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

File: styles/themes/_light.scss [hunk 1]
```diff
@@ -56,6 +56,15 @@ html.light-mode {
...
   }
 
+
+  .custom-navbar-item {
+    .navbar-item {
+      &:hover {
+        color: $k-hovergrey!important;
+      }
+    }
+  }
```
Comment (Jarsen136) [inline styles/themes/_light.scss:66]: IMO, we could remove these lines because they have been written on `styles/layouts/_main-navigation.scss` file.

Comment (prachi00) [reply styles/themes/_light.scss]: moved this code, thanks

File: styles/layouts/_main-navigation.scss [hunk 2]
```diff
@@ -214,6 +219,10 @@
     font-weight: 500;
     cursor: pointer;
     border-radius: 0 !important;
+
+    &:hover {
+      color: $k-hovergrey !important;
+    }
```
Comment (Jarsen136) [inline styles/layouts/_main-navigation.scss:225]: Could we remove these codes? Or it would make something different compared to line121~ line125?

Comment (prachi00) [reply styles/layouts/_main-navigation.scss:225]: yeah that one seems to be a different class

**Comment by exezbcz** [comment]
dark mode button in dropdown changes its text color to k-hovergrey
image

**Comment by prachi00** [comment]
> 

Screen Shot 2022-12-30 at 12 17 19 PM
so this primary should be k-hovergrey?

**Comment by exezbcz** [comment]
> > 
> 
> Screen Shot 2022-12-30 at 12 17 19 PM
> 
> so this primary should be k-hovergrey? @exezbcz

no, look at the screenshot, the text on hover should not change to k-hovergrey and it should stay black, as other buttons
image

**Comment by exezbcz** [comment]
I'm not sure if this was previously the case, but the lines that separate the buttons are now black in darkmode. They should be white.
image
like here: 
image

**Comment by prachi00** [comment]
@exezbcz fixed

**Review by Jarsen136 — APPROVED** [review]
code lgtm

**Comment by yangwao** [comment]
pay 50 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 23.06 USD/KSM ~ 2.168 $KSM
🧗 [EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU](https://kusama.subscan.io/account/EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)
🔗 [0x9d2f005c27759fc0896ab8d58f523855c9f470eb351be841b80778f1fde3e471](https://kusama.subscan.io/extrinsic/0x9d2f005c27759fc0896ab8d58f523855c9f470eb351be841b80778f1fde3e471)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**